blog/zh/blog/2023/10/09/wecity-uses-apisix.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "荷兰智慧城市革命：WeCity 与 APISIX 的合作之旅"
+authors:
+  - name: Arjen Hof
+    title: Author
+    url:
+    image_url:
+  - name: Tim van Densen
+    title: Author
+    url:
+    image_url:
```

Comment:
See https://github.com/apache/apisix-website/pull/1691 , if no `url` or `image url`, delete the variables instead of leaving them in blank.

```suggestion
  - name: Arjen Hof
    title: Author
  - name: Tim van Densen
    title: Author
```